packages/vitest/src/node/error.ts

```diff
@@ -36,10 +36,12 @@ export async function printError(error: unknown, ctx: Vitest) {
       && existsSync(stack.file),
   )
 
+  const errorProperties = getErrorProperties(e)
+
   printErrorMessage(e, ctx.console)
-  await printStack(ctx, stacks, nearest, async (s, pos) => {
+  printStack(ctx, stacks, nearest, errorProperties, (s, pos) => {
     if (s === nearest && nearest) {
-      const sourceCode = await fs.readFile(fileFromParsedStack(nearest), 'utf-8')
+      const sourceCode = readFileSync(fileFromParsedStack(nearest), 'utf-8')
```

Comment:
I found a bug where `fs.readFile` stalls the terminal, and breaks `for of` loop - because of that some stack trace might be missing.